packages/dd-trace/test/exporters/scheduler.spec.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+require('../setup/tap')
```

Comment:
This file can be deleted as it was moved under appsec folder in the RC PR and now does not exist here anymore, thus creating a merge conflict.